tsdef/AudioProcessor.d.ts

```diff
@@ -0,0 +1,65 @@
+import { NoiseCancellationVendor } from './types';
+
+export interface AudioProcessor {
```

Comment:
We should probably export this in `tsdef/index.d.ts`

Reply:
This interface is not public facing - its used internally only.